README.md

```diff
@@ -10,18 +10,20 @@ We welcome and appreciate all contributions. Got questions or want to discuss so
 
 # Venafi Collection for Ansible
 
-This collection, called `machine_identity`, uses Venafi Trust Protection Platform (TPP) or Venafi as a Service (VaaS)
-to provide certificates and keys to Ansible solutions involving machine identities.
+This collection, called `machine_identity`, uses [Venafi Trust Protection Platform](https://www.venafi.com/platform/trust-protection-platform) or [Venafi as a Service](https://www.venafi.com/venaficloud) to provide keys and certificates to Ansible solutions that require machine identity management.
 
 If you like this collection, please give us a rating on [Ansible Galaxy](https://galaxy.ansible.com/venafi/machine_identity).
 
+<!-- DW: what's a collection? Is that different than a solution, the term we used with Terraform? -->
+
```

Comment:
In Ansible, a "collection" (of roles) is more analogous to a "provider" in Terraform (which includes one or more "resources").  Hence the reason we moved to a collection from the role we started with... we needed it to be able to serve multiple use cases including TLS certificate lifecycle, TLS certificate policy management, and SSH certificate.